tests/test_stage.py

```diff
@@ -165,3 +165,27 @@ def test_remote_dependency(self):
         assert main(["import", "remote://storage/file", "movie.txt"]) == 0
 
         assert os.path.exists("movie.txt")
+
+
+class TestIgnore(TestDvc):
```

Comment:
This test is not related to dvc/stages.py, but rather to dvc/repo, so it should be in `tests/test_repo.py`.